src/views/authenticated/List.tsx

```diff
@@ -16,17 +19,47 @@ export function List({ data: unfilteredListItems }: Props) {
 		);
 	}, [searchTerm, unfilteredListItems]);
 
+	const handleAddItemsClick = () => {
+		navigate("/manage-list");
+	};
+
 	return (
 		<>
 			<p>
 				Hello from the <code>/list</code> page!
 			</p>
 
+			{unfilteredListItems.length === 0 && (
+				<div>
+					<h2>Your list is ready!</h2>
+					<h3>
+						It looks like you haven’t added any items yet.
+						<br />
+						Let’s get started by adding your first item.
+					</h3>
+					<button
+						onClick={handleAddItemsClick}
+						aria-label="Navigate to add items to your list"
+					>
+						{"Get started!"}
+					</button>
+				</div>
+			)}
```

Comment:
If the button logic is centralized and the 2 states for when there is no data and is data are still wanted I think this could get changed to a terneary instead of the short eval and this could handled the change in the prompt. It would allow the logic for this to be in one place instead of across the 2 component areas.

```suggestion
			{unfilteredListItems.length === 0 ? (
				<div>
					<h2>Your list is ready!</h2>
					<h3>
						It looks like you haven’t added any items yet.
						<br />
						Let’s get started by adding your first item.
					</h3>
					<button
						onClick={handleAddItemsClick}
						aria-label="Navigate to add items to your list"
					>
						{"Get started!"}
					</button>
				</div>
			) : (
                <h3>Want to add more items to your list?</h3>
            )}
```

Reply:
Great logical thought @bbland1 !!! I like that you guys incorporated semantic HTML with the different headers. I wanted to add on to that by suggesting we replace the <div> elements with a <section> for improved semantic HTML

Reply:
Nice!

I was thinking the other night ,"I've been using divs a lot in this project", but it was late at night and I couldn't remember any other elements to use besides main or body! 🤣 🤣 🤣 🤣 🤣